paymentsheet/src/main/java/com/stripe/android/link/ui/updatecard/UpdateCardScreenViewModel.kt

```diff
@@ -30,6 +65,25 @@ internal class UpdateCardScreenViewModel @Inject constructor(
         logger.info("Cancel button clicked")
     }
 
+    private fun initializeInteractor(
+        cardPaymentDetails: ConsumerPaymentDetails.Card
+    ): EditCardDetailsInteractor = DefaultEditCardDetailsInteractor.Factory().create(
+        coroutineScope = viewModelScope,
+        // Until card brand choice is supported in Link, we don't allow CBC.
+        isCbcModifiable = false,
```

Comment:
We support CBC in the “add payment method” screen. Maybe we can make this more of a todo, since we plan to pick this up as a fast-follow?

Reply:
yeah actually let's include it here. See this commit allowing network preference edits, if available: https://github.com/stripe/stripe-android/pull/10715/commits/574b24e74b5c9680129e9fc13ee8e651144dfa8d.

Next PR will include sending the preferred network when updating payment details.